aptos-move/aptos-release-builder/src/components/mod.rs

```diff
@@ -31,41 +38,90 @@ pub struct ReleaseConfig {
     pub consensus_config: Option<OnChainConsensusConfig>,
 }
 
+fn fetch_and_compare<T: OnChainConfig + PartialEq>(
+    client: &Option<Client>,
+    expected: &T,
+) -> Result<bool> {
```

Comment:
Similarly, `fetch_and_compare`, could you add a comment, or rename it to `fetch_and_equals` or something like that?